sites/website/package.json

```diff
@@ -8,8 +8,9 @@
     "build": "npm run prepare-docs & docusaurus build"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.50",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.50",
+    "@docusaurus/core": "2.0.0-alpha.48",
```

Comment:
why are we going down to 2.0.0-alpha.48?

Reply:
@janechu sorry, just seeing this - there are regressions with the live code editor and a couple other things related to it, this is the best way to get it to work. I was planning on bumping up if a release came out but it hasn't (need 2.0.0-alpha.57 published).